ACPO-model/src/train.py

```diff
@@ -325,14 +330,15 @@ def learn(train_data_path:str, test_data_path:str, log_dir:str, train_kwargs:dic
 
                 if args.save_model:
                     torch.save(model.state_dict(), os.path.join(log_dir, "plu.pt"))
-                    save_pb(model, log_dir, num_features)
+                    # save_pb(model, log_dir, num_features)
+                    save_pb_inline(model, log_dir, num_features)
```

Comment:
Is function name with `inline` inside reasonable?

Reply:
i will change name soon